elizabeth/homework1.js

```diff
@@ -0,0 +1,173 @@
+/* Section 1
+a) True -- == is not exact and JS will auto convert "1" to a number.
+
+b) False -- === is exact, the values must match in type and value.
+
+c) True -- for the same reason "a" was true, both value and type are not checked, however it does not actually return a value.
+
+d) False -- the right side would be "undefined" and x!== undefined
+
+e) False -- empty string is "undefined" and the right side is a string equal to a space.
+
+f) True -- if x has a value of a string or a number, it is truthy, False if x has a value of Nan, empty string, or undefined.
+
+g) True -- x could equal undefined if it's value is ' '
+
+h) False -- string 9 and string 10 cannot be evaluated like numbers
+
+i) False -- typeof x will return a value, adding 1 to that will not ever return just "number"
+
+j) False -- same issue as "i" ^
+
+k) True -- now the typeof is on the expression (x % 2), JS will assume x is a number because of what follows it.
+
+l) False -- the right side would be one more than the left side.
+
+m) True -- the left side would not have returned the + 1 yet.
+
+n) False -- string 1 added to x will not equal number 1 added to x. If x is a number it might convert, but if it is anything else, it would concatonate on the left and not on the right.
+
```

Comment:
Assuming x is a string, this will return true. Remember that we're just looking at a truthiness, similar to question a. The right hand side will still get concatenated.